Form/Type/VihuvacRecaptchaType.php

```diff
@@ -133,14 +133,6 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * {@inheritdoc}
      */
-    public function getParent()
```

Comment:
This function is used by symfony versions ^2.3 in order to create custom field types, e.g: [create a custom field type](http://symfony.com/doc/current/form/create_custom_field_type.html).